QEfficient/generation/text_generation_inference.py

```diff
@@ -18,6 +19,17 @@
 from QEfficient.utils import padding_check_and_fix
 from QEfficient.utils.logging_utils import logger
 
+
+@dataclass
+class CloudAI100ExecInfo:
+    generated_texts: List[str] = None
```

Comment:
Why are you assigning None, can there be a case where one/all will not be passed while initializing this class?

Reply:
It is returned in cloud_ai_100_exec_kv and cloud_ai_100_exec_kv_helper. in both places no params will be None, updating this.